ndmg/stats/qa_reg.py

```diff
@@ -27,13 +27,13 @@
 import re
 import sys
 import numpy as np
+from numpy import *
```

Comment:
absolutely never do `from <package> import *` [ever](https://docs.quantifiedcode.com/python-anti-patterns/maintainability/from_module_import_all_used.html). This is bad software engineering practice and will lead to very annoying and hard-to-find namespace bugs.

call `np.<function>` instead. Delete this line.